.cbmc-batch/jobs/aws_array_list_copy/aws_array_list_copy_harness.c

```diff
@@ -16,61 +16,30 @@
 #include <aws/common/array_list.h>
 #include <proof_helpers/make_common_data_structures.h>
 
-/* These values allow us to reach a higher coverage rate */
-#define MAX_ITEM_SIZE 2
-#define MAX_INITIAL_ITEM_ALLOCATION (UINT64_MAX / MAX_ITEM_SIZE) + 1
-
 /**
- * Runtime: 0m13.098s
+ * Runtime: 32s
```

Comment:
Is it the different proof style or a change in CBMC that has caused this runtime increase?

Reply:
The new improvements in CBMC actually lower the verification time to `10s`. I've updated it now.